en/config/actuators.md

```diff
@@ -1,96 +1,236 @@
+
+![Geometry motor: standard vtol](../../assets/config/actuators/qgc_geometry_standard_vtol_motors.png)
+
+Motors have most of the same configuration fields as for the [multicopter geometry](#motor-geometry-multicopter).
+There is an additional field to indicate the direction in which the motor moves the vehicle (for a standard VTOL, the hover motors are usually set "upwards" and the pusher motor is set to "forwards").
+
+- `Axis`: One of `Upwards`, `Downwards`, `Forwards`, `Backwards`, `Leftwards`, `Rightwards`, `Custom`
+  - If `Custom` is selected, then the UI displays three additional fields for setting the motor orientation.
+
+#### Motor Geometry: Other Vehicles
+
+Other vehicle types will define an appropriate motor geometry for their frame type.
+Once again these motors will generally have the same kinds of properties as shown above.
+
+For example, a fixed-wing vehicle may just have a single pusher moter, while a rover with differential steering will have a motor for throttle and for steering.
+
+#### Bidirectional Motors
+
+Some vehicles may use bidirectional motors (i.e. motor supports spinning in both directions)
 For example, ground vehicles that want to move forwards and backwards, or VTOL vehicles that have pusher motors that go in either direction.
 
 If bidiectional motors are used, make sure to select the **Reversible** checkbox for those motors (the checkbox is displayed as an "advanced" option).
 
-![Reversible](../../assets/config/actuators/qgc_geometry_reversible.png)
+![Reversible](../../assets/config/actuators/qgc_geometry_reversible_marked.png)
 
 Note that you will need to also ensure that the ESC associated with bidirectional motors is configured appropriately (e.g. 3D mode enabled for DShot ESCs, which can be achieved via [DShot commands](../peripherals/dshot.md#commands)).
 
 
+### Control Surfaces Geometry
+
+The control surfaces section of the geometry panel lets you set the number of control surfaces that are present on the vehicle.
+You can then specify the type of each individual surface, and configure the appropriate scaling values for the Roll scale, Yaw scale, Pitch scale, Trim and Slew rate (advanced).
+
+An "example" control surface section for a vehicle with two ailerons is shown below.
+Note that ailerons only affect roll, so the pitch and yaw fields are disabled.
+
+![Control Surface Setup Example](../../assets/config/actuators/control_surfaces_geometry.png)
+
+The fields are:
+
+- `Control Surfaces`: The number of control surfaces (set this first!)
+- `Type`: The type of each control surface: `LeftAileron`, `RightAileron`, `Elevator`, `Rudder`, `Left Elevon`, `Right Elevon`, `Left V-Tail`, `Right V-Tail`, `Left Flap`, `Right Flap`, `Airbrakes`, `Custom`.
+- `Roll scale`: Roll scale (normalised: -1 to 1)
+- `Pitch scale`: Pitch scale (normalised: -1 to 1)
+- `Yaw scale`: Yaw scale (normalised: -1 to 1)
+- `Trim`: ?
+- `Skew`: ?
+- `Lock control surfaces in hover`: `Enabled` or `Disabled`.
```

Comment:
Most VTOLs don't require the control surfaces in hover for control (exception: duo tailsitter that uses the elevons for pitch and yaw control). Sometimes you still want to enable them in hover already, as they can help the MC motors in stabilizing the vehicle in wind or when flying fast.

Reply:
Thanks @sfuhrer ! I guess if they aren't "required" then locking them prevents them moving around an potentially affecting vehicle control?

Is this OK?
```suggestion
- `Lock control surfaces in hover`:
  - `Enabled`: Most vehicles do not use control surfaces in hover. Use this setting to lock them so that they don't affect vehicle dynamics.
 - `Disabled`: Set this for vehicles that use control surfaces in hover, such as the duo tailsitter (which uses elevons for pitch and yaw control). It should also be set for vehicles that use control surfaces to provide additional stabilization in hover mode when moving at speed or in high winds.
```

Reply:
>Thanks @sfuhrer ! I guess if they aren't "required" then locking them prevents them moving around an potentially affecting vehicle control?

Yes correct, they usually do more harm then good. 

And yes, I like your proposal :+1: